### DISCUSSION TITLE
Modbus Device data Issue

### DISCUSSION THREAD

**Discussion by sirisatwika** [edgexfoundry/community#115] [Q&A]
We are using EdgeX Kamakura Version.
We are simulating data for modbus through ModbusPal Simulator.The minimum and Maximum value for a particular device has been set. Device Profile and device  has been added through EdgeX GUI.
But while we try to Fetch the data within the range from EdgeX Foundry, we are getting a random value which is out of range.
Can you please help us with the issue.

**Comment by cloudxxx8** [comment]
Did you set minimum and maximum value in your simulator or device profile?
If it is only set in the device profile, you might still receive the value which is out of range.  The simulator should also be set up properly.

**Reply by sirisatwika** [reply]
Thankyou for your response,

I have set the same minimum and maximum value in the modbus simulator where data is generated and also in the device profile.
